testing/recipes/run_cocodataset_nvidiadl_ssd.sh

```diff
@@ -51,4 +53,5 @@ fi
 
 for i in $(seq $RUN_CNT); do
     ./run_toolbox.py benchmarking run_nvidiadl_ssd "$gpu_node_hostname" --epochs=$EPOCHS --threshold=$THRESHOLD
+    ./run_toolbox.py benchmarking assess_benchmark_stats --min_sample_rate=$MINSR --max_duration=$MAXDR
```

Comment:
I don't think this requires a new role, just write a bash function  doing the same checks as what you do in `assess_benchmark_stats`

and actually, it may be time to add a new script `testing/prow/benchmarking.sh` that will do something similar to this one + the performance assessement

in this script, just let everything else as you've modified here, for easy reexecution